irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -247,19 +247,15 @@ namespace iroha {
       auto transactions_errors_log = validation::TransactionsErrors{};
       auto valid_proto_txs = validateTransactions(
           proposal.transactions(), temporaryWsv, transactions_errors_log, log_);
-      auto validated_proposal = shared_model::proto::ProposalBuilder()
-                                    .createdTime(proposal.createdTime())
-                                    .height(proposal.height())
-                                    .transactions(valid_proto_txs)
-                                    .createdTime(proposal.createdTime())
-                                    .build();
 
-      log_->info("transactions in verified proposal: {}",
-                 validated_proposal.transactions().size());
-      return std::make_pair(std::make_shared<decltype(validated_proposal)>(
-                                validated_proposal.getTransport()),
+      // Since proposal came from ordering gate it was already validated.
+      // All transactions has been validated aswell
+      // This allows for unsafe construction of proposal
+      auto validated_proposal = factory_->unsafeCreateProposal(
+          proposal.height(), proposal.createdTime(), valid_proto_txs);
+
+      return std::make_pair(std::move(validated_proposal),
                             transactions_errors_log);
     }
-
```

Comment:
There is an empty line in the beginning between namespaces and the code, so this one should not be removed

Reply:
I think it's better to remove the empty line at the beginning